### ISSUE TITLE
Unable to silence statement logging in web workers

### ISSUE THREAD

**Issue by stx** [simolus3/moor#2451]
Getting tons of these logs constantly which makes debugging very difficult. `WebDatabase.withStorage()` defaults logStatements to false but they're still logging:

```
[OUT]: Request (id = 4687): StatementMethod.select: SELECT <blah blah blah>
[IN]: SuccessResponse (id = 4687): Instance of 'SelectResult'
```

Web worker:

```
// ignore_for_file: avoid_web_libraries_in_flutter

import 'dart:html';

import 'package:drift/web.dart';
import 'package:drift/web/worker.dart';

void main() {
  WorkerGlobalScope.instance.importScripts('sql-wasm.js');

  driftWorkerMain(() {
    return WebDatabase.withStorage(
      DriftWebStorage.indexedDb(
        'worker',
        migrateFromLocalStorage: false,
        inWebWorker: true,
      ),
    );
  });
}
```

Flutter app:

```
  DatabaseConnection conn = await connectToDriftWorker(kReleaseMode ? 'worker.dart.min.js' : 'worker.dart.js');

  return Database(conn.executor);
```

**Comment by simolus3** [comment]
Yes, this was an unfortunate regression in the `connectToDriftWorker` API. It's fixed on `develop`, but that hasn't been released yet. I'll release this with a few other bugfixes tomorrow and I'll keep the issue open until then.

**Comment by simolus3** [comment]
Fixed in drift 2.8.1
